ols/src/cache/postgres_cache.py

```
@@ -388,32 +388,25 @@ def set_topic_summary(
-            except (psycopg2.OperationalError, psycopg2.InterfaceError):
-                try:
-                    logger.info("Connection error during poll, attempting reconnect")
-                    self.connect()
-                    return True
-                except Exception as e:
-                    logger.warning("Reconnect attempt failed: %s", e)
-                    return False
+                logger.info("Detected dead connection, attempting reconnect")
+                self.connect()
+                return True
+            except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)